.github/workflows/turf.yml

```diff
@@ -22,14 +22,14 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
+        cache: "pnpm"
```

Comment:
^ there's a new version out, so I grabbed that, and you can tell it that we're using pnpm for it to cache things for us automatically. I'm hoping this speeds up the `pnpm install` part of our CI on subsequent runs.